package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "lint": "solium --dir contracts && eslint --fix migrations/ test/",
     "test": "truffle test --network develop",
-    "coverage": "solidity-coverage"
+    "coverage": "solidity-coverage",
+    "flattener": "truffle-flattener"
```

Comment:
Since introduction of `npx` I don't think that exposing command line tools this way is strictly necessary.